domain-models-runtime-it/src/test/java/org/folio/rest/WrappedBooleanApiTest.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.rest;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+
+import org.junit.jupiter.api.Test;
+
+public class WrappedBooleanApiTest extends ApiTestBase {
```

Comment:
Can you remove `public`? Sonar complains about it: https://sonarcloud.io/project/issues?id=org.folio%3Araml-module-builder&pullRequest=698&resolved=false&types=CODE_SMELL